apps/src/applab/designElements/elementUtils.js

```diff
@@ -234,15 +234,17 @@ export function setDefaultBorderStyles(element, options = {}) {
  * @param {string} cssPaddingString value from element.style.padding
  */
 export function calculatePadding(cssPaddingString) {
-  const paddingRegEx = /\s*?(\d*)(px)?\s*(\d*)?(px)?\s*(\d*)?(px)?\s*(\d*)?(px)?\s*?/i;
-  const paddingResult = paddingRegEx.exec(cssPaddingString);
+  // Extract up to 4 numbers, ignoring the 'px' that may be included
 
-  const paddingValues = [
-    parseInt(paddingResult[1], 10),
-    parseInt(paddingResult[3], 10),
-    parseInt(paddingResult[5], 10),
-    parseInt(paddingResult[7], 10)
-  ];
+  // NOTE: if other measurement values (e.g. 'em') make it into the padding string,
+  // we will treat them as 'px' values
+
+  // Ideally, we could use getComputedStyle(), but we need to work with
+  // detached DOM nodes, which doesn't work on Chrome and Safari
+
+  const paddingValues = (cssPaddingString || '')
+    .split(/\s+/)
+    .map(part => parseInt(part, 10));
   for (
     var validPaddingValues = 0;
```

Comment:
Should be validPaddingValueCount or something similar. The current name makes it sound like this is a list.

Reply:
It is an array, so I think the name makes sense.

Reply:
Sorry for the confusion, this was referring to 'validPaddingValues'
(should have been a nit comment)